content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -222,6 +227,83 @@ At this point, the flow is now complete.
+
+.. note::
+   A 100% down payment is **not** the same as a full payment of the sales order.
+
+   A sales order paid through the regular invoice process will not allow any additional invoices to
+   be generated, and **will not** display the *Create invoice* button on the Sales Order.
+
+   Following this example **will** cause the *Create invoice* button to be displayed on the Sales
+   Order. This is because Odoo expects another invoice to be created after the down payment
+   to complete payment of the sales order.
+
+
+The *Solar Panel Installation* product is being used in this example.
+
+To configure a 100% down payment, begin by navigating to :menuselection:`Sales app --> New`, and add
+a :guilabel:`Customer` to the quote.
+
+Next, click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab, and select the
+:guilabel:`Solar Panel Installation` product.
+
+Upon clicking the :guilabel:`Confirm` button, the quotation turns into a sales order. At that point,
+an invoice can now be created by clicking :guilabel:`Create Invoice` in the top-left corner.
+
+.. image:: down_payment/100p-order-confirmed.png
+   :align: center
+   :alt: Confirmed sales order for the Solar Panel Installation product.
+
+On the :guilabel:`Create invoices` pop-up window that appears, select :guilabel:`Down payment
+(percentage)`, and type `100` in the :guilabel:`Down Payment Amount` field. Then, if desired, select
+an :guilabel:`Income Account' and :guilabel:`Customer Taxes` in their respective fields.
+
+.. image:: down_payment/100p-down-payment-percentage.png
+   :align: center
+   :alt: The Down payment (percentage) option selected with 100% set as the Down Payment.
+
+Next, click :guilabel:`Create Draft Invoice` to create an invoice draft. This will also
+bring the draft invoice into view, which includes the :guilabel:`Down partment`as a
+:guilabel:`Product` in the :guilabel:`Invoice Lines` tab.
+
+The invoice can now be confirmed and posted by clicking :guilabel:`Confirm`. Confirming the
+invoice changes the status from :guilabel:`Draft` to :guilabel:`Posted`. It also reveals a new
+series of buttons at the top of the page.
+
+.. image:: down_payment/100p-confirm-invoice-draft.png
+   :align: center
+   :alt: A sample draft invoice with down payment mentioned in Odoo Sales.
+
+The payment can be registered by clicking the :guilabel:`Register Payment` button.
+
+.. image:: down_payment/100p-register-payment.png
+   :align: center
+   :alt: Showcase of the Register Payment button on a confirmed customer invoice.
+
+Doing so reveals a :guilabel:`Register Payment` pop-up form, which is auto-populated with the
+necessary information. Confirm the information provided is correct and make any necessary
+adjustments. When ready, click the :guilabel:`Create Payment` button.
+
+.. image:: down_payment/100p-create-payment.png
+   :align: center
+   :alt: Showcase of the Register Payment pop-up window with create payment button.
```

Comment:
Same reasoning as above, your written instructions are very clear– these images can be removed IMO :)
```suggestion

The payment can be registered by clicking the :guilabel:`Register Payment` button.

Doing so reveals a :guilabel:`Register Payment` pop-up form, which is auto-populated with the
necessary information. Confirm the information provided is correct and make any necessary
adjustments. When ready, click the :guilabel:`Create Payment` button.

```